kube/Cargo.toml

```diff
@@ -24,6 +24,9 @@ jsonpatch = ["json-patch"]
 ws = ["tokio-tungstenite"]
 oauth = ["tame-oauth"]
 gzip = ["async-compression"]
+proxy = []
+proxy-native-tls = ["native-tls", "proxy", "hyper-proxy/tls"]
+proxy-rustls-tls = ["rustls-tls", "proxy", "hyper-proxy/rustls-base"]
```

Comment:
I guess the want for `weak-dep-features` returns :-)

Reply:
Yeah... When it becomes stable, we can have just `proxy` feature to enable `hyper-proxy` and include its features in our tls features.

I removed `proxy` feature for now because it's redundant. Proxy is enabled with `proxy-native-tls` or `proxy-rustls-tls`.